specification/cognitiveservices/data-plane/ContentModerator/v1.0/examples/GetFramesResource.JSON

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "baseUrl": "southeastasia.api.cognitive.microsoft.com",
+    "teamName": "Content moderator",
+    "reviewId": "201711v18ea829372b14f9e9d382621e62429a9",
+    "startSeed": "1",
+    "noOfRecords": "1"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+      },
+      "body": {
+        "reviewId": "201711v18ea829372b14f9e9d382621e62429a9",
+        "videoFrames": [
+          {
+            "timestamp": "1122",
+            "frameImage": "https://rtsvccontentwu.blob.core.windows.net/580040/FRM_201711v18ea829372b14f9e9d382621e62429a9_1122.jpg",
+            "metadata": [
+              {
+                "key": "apiScoreVal",
+                "value": "0.206"
+              },
+              {
+                "key": "a",
+                "value": "False"
+              }
+            ],
+            "reviewerResultTags": [],
+            "frameImageBytes": ""
```

Comment:
You need to remove this as well.

Reply:
Removed this parameter in the latest update. Thanks.